snmp/tests/test_profiles.py

```diff
@@ -989,24 +1206,160 @@ def test_dell_poweredge(aggregator):
                 'snmp.{}'.format(count), metric_type=aggregator.MONOTONIC_COUNT, tags=tags, count=1
             )
 
-    # IDRAC
     indexes = ['26', '29']
     for index in indexes:
         tags = ['chassis_index:{}'.format(index)] + common_tags
-        for gauge in SYSTEM_STATUS_GAUGES:
+        for gauge in POWEREDGE_SYSTEM_STATUS_GAUGES:
             aggregator.assert_metric('snmp.{}'.format(gauge), metric_type=aggregator.GAUGE, tags=tags, count=1)
-    powers = ['supply1', 'supply2']
-    for power in powers:
-        tags = ['supply_name:{}'.format(power)] + common_tags
-        aggregator.assert_metric('snmp.enclosurePowerSupplyState', metric_type=aggregator.GAUGE, tags=tags, count=1)
```

Comment:
Just to confirm, `enclosurePowerSupplyState` it's not available for dell poweredge ?

Seems not present in https://www.circitor.fr/Mibs/Mib/M/MIB-Dell-10892.mib

Reply:
Indeed, not implemented by `MIB-Dell-10892`

Reply:
And could not find those metrics within the SNMP walk from Poweredge R640